lib/thrift/generator.ex

```diff
@@ -1,6 +1,7 @@
 defmodule Thrift.Generator do
   alias Thrift.Parser.FileGroup
   alias Thrift.Generator.EnumGenerator
+  alias Thrift.Generator.Service, as: ServiceGenerator
```

Comment:
I was going to mention that you're repeating Generator there. Idiomatically, they should be Thrift.Generator.Enum, Thrift.Generator.Struct, etc.